### ISSUE TITLE
Repository URL not available

### ISSUE THREAD

**Issue by julienrbrt** [leighmcculloch/vangen#19]
We are using vangen for https://github.com/cosmos/vanity, works great thanks!
However, the repository URL doesn't show up in pkg.go.dev:

![image](https://github.com/user-attachments/assets/2fb5f3bf-e6eb-4637-8f90-8145bb3a6ddc)

**Comment by leighmcculloch** [comment]
Hi @julienrbrt, thanks for reporting.

It looks like the vanity page (e.g. https://cosmossdk.io/api/) has meta:
```html
<meta name="go-import" content="cosmossdk.io/ git https://github.com/cosmos/cosmos-sdk">
<meta name="go-source" content="cosmossdk.io/ https://github.com/cosmos/cosmos-sdk https://github.com/cosmos/cosmos-sdk/tree/main{/dir} https://github.com/cosmos/cosmos-sdk/blob/main{/dir}/{file}#L{line}">
```

The thing that seems odd is that for both the `go-import` and `go-source` the leading import path is supposed to be `cosmossdk.io/api` not `cosmossdk.io/`.

**Comment by leighmcculloch** [comment]
@julienrbrt I've pushed a fix in https://github.com/leighmcculloch/vangen/pull/20. Are you able to try regenerating your vanity page using it, and then try getting pkg.go.dev to refresh your package to confirm if that resolves the issue?

**Comment by leighmcculloch** [comment]
Excellent! I've merged the fix and released it in v1.4.1. Thanks again for reporting!
